### DISCUSSION TITLE
Improve couple of things

### DISCUSSION THREAD

**Discussion by JohnMaverick** [YetiForceCompany/YetiForceCRM#15466] [Feature improvement]
I found this https://github.com/YetiForceCompany/YetiForceCRM/issues/15460 so I throwing this:
CRM system should be designed with keyboard operations in mind. When operator/marketing man needs to fix lots of records everyday, clicking is annoying. If system is optmised for both pointer and keybord it will be easier to deploy keyboard shortcuts, which are life saver in many scenarios.

**Comment by mariuszkrzaczkowski** [comment]
suggest a specific solution, 
there is a possibility of keyboard shortcuts in the menu

**Comment by JohnMaverick** [comment]
In List view:
  Page Up/Page Down to page through the listing
  Double press any key to jump to that letter in the view
  ctrl+a to add a new item to the list

In Detail view:
-  ctrl+e to edit
-  ctrl+d to duplicate
-  Page up/Page down to navigate through the related items
-  right/left to move to the next or previous entry

In Edit view
-  ctrl+s to save

Everywhere
-  ctrl+l to go to the list view
-  escape to go back to the previous screen

**Reply by mariuszkrzaczkowski** [reply]
Added https://github.com/YetiForceCompany/YetiForceCRM/commit/916036f00d77a445e41dc2c6f2468c16a8cfccf3

List view:
ALT + A - to add a new item to the list

Detail view:
ALT + E - go to edit
ALT + D - go to duplicate
ALT + L - go to the list view

Edit view

ALT + S - go to save

**Reply by mariuszkrzaczkowski** [reply]
https://yetiforce.com/en/knowledge-base/documentation/user-documentation/item/keyboard-shortcuts

**Reply by s2b-git** [reply]
Morning

Merged changes as per commit https://github.com/YetiForceCompany/YetiForceCRM/commit/916036f00d77a445e41dc2c6f2468c16a8cfccf3 but shortcuts does not triggering. Tried in incognito mode - the same. Are there any other dependencies? Running 6.2

**Reply by rskrzypczak** [reply]
Disable loading min.js files in config/Developer.php
public static $MINIMIZE_JS = false;

**Reply by s2b-git** [reply]
Disabled but cannot save record. System stucks on saving. Checked log but nothing showing up

**Reply by ArniSamano** [reply]
Marius, navigation Left/Right shortcuts are not working. As detail view doesn't have navigation buttons, this will help a lot

**Reply by s2b-git** [reply]
Missing those navigation badly too. One you get to detail view you stuck with one or two clicks more and loosing focus
:)
